ambuda/templates/proofing/pages/edit.html

```diff
@@ -166,7 +166,5 @@ <h1 class="text">
 
 <script src="https://cdn.jsdelivr.net/npm/openseadragon@3.1/build/openseadragon/openseadragon.min.js"></script>
 {% set image_url = url_for("site.page_image", project_slug=project.slug, page_slug=cur.slug) %}
-<script type="text/javascript">
-const IMAGE_URL = "{{ image_url }}";
-</script>
+<script type="application/json" id="image_url">"{{ image_url }}"</script>
```

Comment:
I made this change so that we do not have to use `unsafe-inline` for the JS CSP.

Reply:
Unfortunately there are still a few other places where we have small amounts of inline JS, such as `ambuda/templates/proofing/create-project-post.html`. But, that's all I could find with a quick `grep`.

I think bits of inline JS give us the flexibility to ship some lightweight behavior without needing to be part of our JS bundle. Can you think of a way to retain that flexibility here?

Reply:
We _could_ use hashes / nonces to allow inline JS, but I think it's better to just not use inline JS at all, so we can get the XSS protection. I can change it for this file, looks like the only remaining place with inline JS. I think it's fine to ship lightweight behavior by adding code to our JS bundle -- it's still pretty flexible -- it might actually be the most ideal way to do it instead of adding inline JS `<script>` blocks in different pages.